introduction.md

```diff
@@ -1,2 +1,8 @@
 Hi I'm Bryan
-JavaScript is awesome.	
+Javascript is awesome.	
+
+Deadly skills
+
+* Git
+* JS
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?